src/Extensions/CMSMainExtension.php

```diff
@@ -5,15 +5,12 @@
 use SilverStripe\Core\Extension;
 use SilverStripe\View\Requirements;
 
+/**
+ * @deprecated 1.2 This extension no longer serves a purpose. Write a standalone extensions instead.
+ */
 class CMSMainExtension extends Extension
```

Comment:
I left this extension because it'd be breaking to remove it - assuming someone might've `MyExtension extends CMSMainExtension`.